perun-base/src/main/java/cz/metacentrum/perun/core/api/exceptions/AlreadyAdminException.java

```diff
@@ -3,7 +3,7 @@
 import cz.metacentrum.perun.core.api.*;
 
 /**
- * Should be named AlreadyInRoleException.
+ * Thrown when trying to assign role to a user in which the user already is
```

Comment:
I would prefer something like this:
`Thrown when trying to assign role to a user while the user already has the role.`

Reply:
Also, the role can be assigned to a group instead of a user.